packages/client/hmi-client/src/types/ResponsiveMatrix.ts

```diff
@@ -0,0 +1,50 @@
+import { Texture, Resource } from 'pixi.js';
+
+export type SelectedCell = [number, number, number, number];
+
+export enum SelectedCellValue {
+	START_ROW,
+	START_COL,
+	END_ROW,
+	END_COL
+}
+
+export enum CellType {
+	BAR = 'ResponsiveCellBarContainer',
+	LINE = 'ResponsiveCellLineContainer'
+}
+
+export enum CellStatus {
+	NONE,
+	SELECTED
+}
+
+export type Uniforms = {
```

Comment:
Could you add a comment explaining (or a link that explains) what are `Uniforms`.

Reply:
Well if you want to go down that path, there are:
- attribute
- varying
- uniform
- ... maybe something else ???

They are basically all variables that are passed to the shader, with slightly different behaviours and context.

Reply:
do we want to do this, these are all universally understood gl terms, i am not sure we want to embed a gl tutorial in the responsive matrix code. if we still want to do something like this i am more open to putting links in.

Reply:
I tend to agree with Manfred here, let's leave it alone and maybe add a note to the technical-specification that we use WebGL and maybe link people here (https://developer.mozilla.org/en-US/docs/Web/API/WebGL_API).

Reply:
Okay, we need to add this in the README.
I don't like reading code, and I have to be the one figuring out what things are. Either a comment block at the top should announce that this is using WebGL, and refer people to that.